2022-04-25 13:41:29,625 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-25 13:45:21,593 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-25 13:46:28,426 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-25 13:54:16,839 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-25 13:58:49,255 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-25 14:03:21,554 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-25 14:12:37,498 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-25 14:46:52,293 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-25 14:47:03,612 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-25 14:48:00,806 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-25 14:48:06,595 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:46]
2022-04-28 03:10:20,813 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:48]
2022-04-28 03:12:10,921 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:48]
2022-04-28 03:39:32,826 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:48]
2022-04-28 04:14:37,792 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:48]
2022-04-28 04:21:04,208 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:48]
2022-04-28 04:30:55,254 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:48]
2022-04-28 04:47:12,373 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:48]
2022-04-28 04:47:44,884 ERROR: Exception on /reset_password_request [POST] [in /home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/app.py:1455]
Traceback (most recent call last):
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/lordrambock/Python_Projects/Flask/microblog/app/routes.py", line 48, in reset_password_request
    send_password_reset_email(user)
  File "/home/lordrambock/Python_Projects/Flask/microblog/app/email.py", line 18, in send_password_reset_email
    text_body=render_template('email/reset_password.txt',
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/templating.py", line 149, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/jinja2/environment.py", line 1071, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/jinja2/environment.py", line 1000, in get_template
    return self._load_template(name, globals)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/jinja2/environment.py", line 959, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/templating.py", line 59, in get_source
    return self._get_source_fast(environment, template)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/templating.py", line 95, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: email/reset_password.txt
2022-04-28 04:50:46,638 INFO: Microblog startup [in /home/lordrambock/Python_Projects/Flask/microblog/app/__init__.py:48]
2022-04-28 04:55:35,933 ERROR: Exception on /reset_password/eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyZXNldF9wYXNzd29yZCI6MiwiZXhwIjoxNjUxMTQ3Mjk1LjA5OTY0Nn0.xnSfdJ89ipHILah6TI3YWIoOPJzKuXvYewm4L67reIU [GET] [in /home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/app.py:1455]
Traceback (most recent call last):
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/lordrambock/Python_Projects/Flask/microblog/app/routes.py", line 37, in reset_password
    return render_template('reset_password.html', form=form)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/flask/templating.py", line 149, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/jinja2/environment.py", line 1071, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/jinja2/environment.py", line 1000, in get_template
    return self._load_template(name, globals)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/jinja2/environment.py", line 959, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/jinja2/loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/jinja2/environment.py", line 758, in compile
    self.handle_exception(source=source_hint)
  File "/home/lordrambock/Python_Projects/Flask/microblog/venv/lib/python3.9/site-packages/jinja2/environment.py", line 926, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/lordrambock/Python_Projects/Flask/microblog/app/templates/reset_password.html", line 24, in template
    {% endblock %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endblock'.
